### PR TITLE
Added DynamoDB state store doc

### PR DESCRIPTION
**Pull request by trondhindenes**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description
Adds doc page for AWS DynamoDB state store component

## Issue reference
https://github.com/dapr/docs/issues/1216

### CODE REVIEW

**Comment by yaron2** [comment]
hey @trondhindenes can you open this against the v1.0 branch? the rc branches/versions will be removed when 1.0 is released

**Comment by msfussell** [comment]
Also @trondhindenes - In the v1.0 docs this table layout has changed a little, so please pull from the v1.0 branch. See this PR https://github.com/dapr/docs/pull/1213

**Comment by trondhindenes** [comment]
willl do!

**Comment by trondhindenes** [comment]
@msfussell I believe it should be in good shape now.

File: daprdocs/content/en/operations/components/setup-state-store/supported-state-stores/setup-dynamodb.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+type: docs
+title: "DynamoDB"
```
Comment (mukundansundar) [inline daprdocs/content/en/operations/components/setup-state-store/supported-state-stores/setup-dynamodb.md]:
```suggestion
title: "AWS DynamoDB"
```

File: daprdocs/content/en/operations/components/setup-state-store/supported-state-stores/setup-dynamodb.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+linkTitle: "DynamoDB"
```
Comment (mukundansundar) [inline daprdocs/content/en/operations/components/setup-state-store/supported-state-stores/setup-dynamodb.md]:
```suggestion
linkTitle: "AWS DynamoDB"
```

**Review by mukundansundar — COMMENTED** [review]
lgtm. small changes.

**Comment by trondhindenes** [comment]
thanks @mukundansundar - I applied your suggestions

File: daprdocs/content/en/operations/components/setup-state-store/supported-state-stores/_index.md [hunk 3]
```diff
@@ -25,6 +25,12 @@ no_list: true
 | RethinkDB                                                      | ✅   | ✅                  | ✅    | Alpha  |
 | [Zookeeper]({{< ref setup-zookeeper.md >}})                    | ✅   | ❌                  | ✅    | Alpha  |
 
+
```
Comment (mukundansundar) [inline daprdocs/content/en/operations/components/setup-state-store/supported-state-stores/_index.md:28]: @trondhindenes one more change. repeated in line 48.

**Review by yaron2 — APPROVED** [review]
lgtm

@orizohar please review also

File: daprdocs/content/en/operations/components/setup-state-store/supported-state-stores/_index.md (hunk 3 above)
Comment (yaron2) [reply daprdocs/content/en/operations/components/setup-state-store/supported-state-stores/_index.md:28]: I fixed that

**Review by yaron2 — APPROVED** [review]

**Review by mukundansundar — APPROVED** [review]
lgtm

**Review by orizohar — APPROVED** [review]

**Pull request merged**